cli/src/main.rs

```diff
@@ -246,11 +246,11 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         Arg::with_name("json_rpc_url")
             .short("u")
             .long("url")
-            .value_name("URL")
+            .value_name("URL_OR_MONIKER")
             .takes_value(true)
             .global(true)
-            .validator(is_url)
-            .help("JSON RPC URL for the solana cluster"),
+            .validator(is_url_or_moniker)
+            .help("JSON RPC URL or some monikers (\"mainnet-beta\", \"testnet\", \"devnet\", \"localnet\")"),
```

Comment:
If I mindlessly mapped from `ClusterType`, I would choose `development` for `localnet`. But, `--url development` looks odd to me. So, I picked `localnet` from the program development world. Thus, I also introduced new category term for these: `monikers`.